app/js/models/todo.js

```diff
@@ -8,7 +8,7 @@ define([
     },
 
     changeCompleted: function () {
-      this.completed = !this.get('completed');
+      this.set({completed: !this.get('completed')});
```

Comment:
Out of principle I don't believe model's should hold this type of businsess logic, simple methods like this give birth to 1000 lines models.